rules/framework.bzl

```diff
@@ -155,6 +157,9 @@ def _apple_framework_packaging_impl(ctx):
             has_header = False
             for hdr in dep[apple_common.Objc].direct_headers:
                 if hdr.path.endswith((".h", ".hh")):
+                    if hdr.path.endswith("Bridging-Header.h"):
```

Comment:
@segiddins This approach is still not working for me, where am I missing something here?

Reply:
it's propagating the copied path, not the new one